schemas/JoinedRoom.json

```diff
@@ -6,7 +6,7 @@
       "enum": ["JoinedRoom"]
     },
     "trigger": {
-     "description": "The reason for the room change if known.",
+     "description": "The reason for the room being joined if known.",
```

Comment:
I think `s/reason/trigger/` would be saner, reason seems more of a social question